src/main/java/ui/UserInterface.java

```diff
@@ -0,0 +1,66 @@
+package ui;
+
+import tasks.Task;
+
+import java.util.Scanner;
+
+public class UserInterface {
+    private String openingMessage = "Hello! I'm Stille\n" + "What can I do for you?\n";
```

Comment:
Assuming these string literals are constants, perhaps could consider using uppercase with underscores notation (e.g. `OPENING_MSG`, or `LINE_DIVIDER`) to make it more readable!

Reply:
Thank you for the help, implemented all of these!